includes/js/myPluginsTools.js

```diff
@@ -1,151 +1,177 @@
-           
-            const pluginsData = response?.entitlements?.filter(data => data?.type === 'plugin' );
-            const installedPlugins = Object.keys( nfdPluginDetails?.installed )
-            const pluginWithStatus = pluginsData.map(val => ({
-              ...val,
-              isInstalled: installedPlugins?.includes( val.basename ),
-              isActive: Object.values( nfdPluginDetails?.active ).find( ( plugin ) => plugin === val.basename )
-            }))
-            const sortedPluginNames = pluginWithStatus.sort( ( a, b ) => a.name.localeCompare( b.name ) )
-            this.setUpContainer( sortedPluginNames );
-        })
-    
-    })
-        
-    }
+	activate_plugin( plugin_path, href ) {
+		fetch( nfdplugin.restApiUrl + '/newfold-solutions/v1/activate_plugin', {
+			credentials: 'same-origin',
+			method: 'POST',
+			headers: {
+				'Content-Type': 'application/json',
+				'X-WP-Nonce': nfdplugin.restApiNonce,
+			},
+			body: JSON.stringify( {
+				plugin: plugin_path,
+			} ),
+		} )
+			.then( ( response ) => response.json() )
+			.then( ( response ) => {
+				if ( response?.message ) {
+					window.location.href = href;
+				}
+			} );
+	}
 
-    activate_plugin ( plugin_path, href ){
-      fetch(
-        nfdplugin.restApiUrl +
-          '/newfold-solutions/v1/activate_plugin',
-        {
-          credentials: 'same-origin',
-          method: "POST",
-          headers: {
-            'Content-Type': 'application/json',
-            'X-WP-Nonce': nfdplugin.restApiNonce,
-          },
-          body: JSON.stringify({
-            plugin: plugin_path
-          })
-        }
-      ).then((response) => response.json() ).then((response) => {
-        if( response?.message ){
-          window.location.href = href;
-        }
-      })
-       
-    }
+	renderNameAsClass( name ) {
+		return name
+			.replaceAll( ' & ', '-' )
+			.replaceAll( ' ', '-' )
+			.replaceAll( '&', '-' )
+			.toLowerCase();
+	}
 
-    renderCTAUrl( url ) {
-      if ( ! window.NewfoldRuntime || ! window.NewfoldRuntime.siteUrl ) {
-        // fallback to site relative url if no base_url is found
-        return url.replace( '{siteUrl}', '' );
-      }
-      return url.replace( '{siteUrl}', window.NewfoldRuntime.siteUrl );
-    }
+	renderCTAUrl( url ) {
+		if ( ! window.NewfoldRuntime || ! window.NewfoldRuntime.siteUrl ) {
+			// fallback to site relative url if no base_url is found
+			return url.replace( '{siteUrl}', '' );
+		}
+		return url.replace( '{siteUrl}', window.NewfoldRuntime.siteUrl );
+	}
 
-    getElementByStatus( isActive, isInstalled, pluginData ) {
-      if( isActive && isInstalled ){ // active and installed
-        return `<a 
-          title="Manage"
-          href="${this.renderCTAUrl(pluginData?.cta?.url)}"
-          class="nfd-solutions-availble-list-item-button"
-        >${pluginData?.cta?.text}</a>`;
-      } else if ( isInstalled ){ // already installed
```

Comment:
I removed this isActive but not isInstalled section, and instead use the later conditionals to just check if it is a premium/pls or free plugin. Since those are the more pertinent deciders of what the markup should be. I did file a ticket with the installer module though, since if a plugin is installed and not active, the installer opens up and fails to install since the directory already exists. This change doesn't change the issue, but simplifies the logic in rendering these buttons.